src/features/projectsV2/ProjectDetails/components/microComponents/OtherInterventionMetaData.tsx

```diff
@@ -0,0 +1,84 @@
+import styles from '../../styles/PlantLocationInfo.module.scss';
+import { Metadata } from '../../../../common/types/plantLocation';
+
+interface Props {
+  metaData: Metadata | undefined;
+  type: string | undefined,
+  plantDate: string | null | undefined;
+}
+
+const OtherInterventionMetaData = ({ metaData }: Props) => {
+
+  function isJsonString(str: string) {
+    try {
+      const parsed = JSON.parse(str);
+      return typeof parsed === 'object' && parsed !== null;
+    } catch (e) {
+      return false;
+    }
+  }
+
+
+
+  interface MetaDataValue {
+    value: string;
+    label: string;
+  }
+  
+  interface PublicMetaData {
+    [key: string]: string | MetaDataValue;
+  }
+  
+  const renderMetaData = () => {
+    const checkForPublic: { value: string; key: string }[] = [];
+    const parsedData = metaData;
+  
+    if (parsedData?.public && typeof parsedData.public === 'object' && !Array.isArray(parsedData.public)) {
+      Object.entries(parsedData.public as PublicMetaData).forEach(([key, value]) => {
+        if (key !== 'isEntireSite') {
+          if (typeof value === 'string') {
```

Comment:
When the only key in public meta data is `isEntireSite`, then we see a blank slide containing OtherInterventionMetaData.